src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift

```diff
@@ -688,6 +688,24 @@ enum AppStrings {
 		static let riskStatusLoweredAlertPrimaryButtonTitle = NSLocalizedString("Home_Alert_RiskStatusLowered_PrimaryButtonTitle", comment: "")
 	}
 
+	enum ContactDiaryInformation {
```

Comment:
Same as with the stringsFile, there is a section for ContactDiary :)

Reply:
Moved as well